src/routes/Profile/WorkHistory/modals/EditToimenkuvaModal.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sonarjs/no-duplicate-string */
```

Comment:
Should we have these urls as consts or something to avoid the eslint disables? Not just this file, but other places as well..

Urls are not changing too often, but should it be something like:
```tsx 
const URL = '/api/profiili/tyopaikat/{id}/toimenkuvat/{toimenkuvaId}';
const BASE_URL = '/api/profiili/tyopaikat/';
// or whatever suits the current case best
```